### DISCUSSION TITLE
Mouse scrolling support

### DISCUSSION THREAD

**Discussion by juarezr** [Schneegans/Fly-Pie#306] [Ideas & Feature Discussions]
What about using the scrolling wheel button of the mouse to:
1. Opening the first child item when none is selected/hovered.
2. Rotate the selection/hovering and open the second child item on the next tick of the scroll wheel
3. Continue to rotate/select while scrolling the wheel
4. Open the grandchild/next level on the left button click

Sounds good?

**Comment by Schneegans** [comment]
Hi there, thanks for the suggestion! Yet I think that this kind of interaction would be waaay slower than directly selecting an item. Or do you think that this could be faster? If so, why?

Besides, I am currently focusing on the development of the successor project to Fly-Pie, [Kando](https://ko-fi.com/post/Introducing-Ken-Do-L3L7L0FQ2). So I'll not implement new features for Fly-Pie as of now. (I have not much spare-time to spent :cry:).
